Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html

```diff
@@ -1,145 +1,114 @@
+x --> y, y --> z Therefore x --> z
+
+When running a reasoner the combination of these becomes undecideable because a functional property must only be only be obtained between one individual and another,
+however, if it is applied transitively it whill carry over to subsequent pairs who are also connected by this same object property.
+
+
+3.) SYMMETRY + A-SYMMETRY = UNSATISFIABLE 
+
+Symmetry declares that if (x) is related to (y) by object_property_1, then (y) is related to (x) by virtue of same object_property_1
+Ex: Symmtric_Object_Property (sister)
+    Object_Property_Assertion (sister, Mary, Jen)  = "Mary is sister of Jen"
+    It follows that "Jen is sister of Mary" because the relation is symmetrically held between both instances. 
+
+Asymmetry provides that if (x) is related to (y) by object_property_3 then (y) CANNOT relate to (x) by object_property_3
+Ex: Asymmetric_Object_Property (Parent_of)
+    Object_Property_Assertion (parent_of, Vivian, Chris)  = "Vivian is a parent of Chris"
+    It holds that Chris CANNOT be a parent of Vivian. 
+
+The joint of these leads to unsatisfiability because an object property cannot relate mutually between instances while also only applying one direction between one to the other.
+Thus, the result will always be false because both cannot be achieved simulataneously. 
+
+
+4.) A-SYMMETRY + TRANSITIVITY = UNDECIDEABLE
+
+Asymmetry provides that if (x) is related to (y) by object_property_3 then (y) CANNOT relate to (x) by object_property_3
+Ex: Asymmetric_Object_Property (Parent_of)
+    Object_Property_Assertion (parent_of, Vivian, Chris)  = "Vivian is a parent of Chris"
+    It holds that Chris CANNOT be a parent of Vivian. 
+ 
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+If there is an object property that holds as both asymmetrical and transitive, it is possible that an instance (y) could relate to (x) as both bearer and recipient
+of some object property if this object property is shared between other instances and (x). Thus the reasoner could not decide between whom it holds. 
+
+
+
+5.) TRANSITIVE + IRREFLEXIVE = UNDECIDEABLE
+
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+Irreflexive hold that no individual can be connected by such an object property to itself.
+
+Syllogism: 
+
+Transitive Property Assertion: brother_of
+Irreflexive Property assertion: brother_of 
+
+    x is brother of y
+    y is brother of z
+    z is brother of x
+Therefore, x is brother to itself. 
+
+This leads to a contradiction because x cannot be brother to itself by virtue of irreflexivity. 
```

Comment:
Let R be a transitive property and irreflexive. Transitivity implies that if xRy and yRz, then xRz. Irreflexivity, on the other hand, prohibits any reflexive relationships (i.e., xRx is forbidden). In a scenario where the transitive property causes x to relate to itself (e.g., xRy, yRz, xRz), this creates a contradiction with irreflexivity, leading to undecidability

Reply:
@elenamiliv Almost, but I wouldn't say the contradiction results in undecidability. You can leave it at contradiction ;)